src/apis.js

```diff
@@ -55,7 +56,7 @@ async function transfer (request: {|
   sendMessage: ?string,
   cryptoType: string,
   data: string,
-  sendTxHash: TxHash,
+  sendTxHash: ?TxHash,
```

Comment:
In what cases do we have a new transfer without a txHash?

Reply:
backup